src/transactions.ts

```diff
@@ -77,6 +78,34 @@ export function createSelfNominateTransaction(
     });
 }
 
+export function createReportDoubleVoteTransaction(
+    sdk: SDK,
+    message1: RLP.Input,
+    message2: RLP.Input
+): Custom {
+    return sdk.core.createCustomTransaction({
+        handlerId: HANDLER_ID,
+        bytes: RLP.encode([REPORT_DOUBLE_VOTE_ACTION_ID, message1, message2])
```

Comment:
messsage1, message2 shouldn't be encoded as `string` in RLP. They should be encoded as a `list`

Reply:
Added a new type for `ConsensusMessage`.